packages/components/src/color-control/index.native.js

```diff
@@ -23,6 +24,7 @@ function ColorControl( {
 			aria-describedby={ !! help ? id + '__help' : undefined }
 			onPress={ onPress }
 			color={ color }
+			withColorIndicator={ withColorIndicator }
```

Comment:
Is it intentional just to show that this props could be passed? It would be easily passed as a part of `{...props}`

Reply:
Yup, it's intentional to present which props _exaclty_ that component accepts.